packages/pluggableWidgets/datagrid-date-filter-web/src/components/__tests__/DatePicker.spec.tsx

```diff
@@ -88,7 +89,45 @@ describe("Date picker component", () => {
             // Trick to trigger events (Should be dd/MM/yyyy - dd/MM/yyyy) but the library does not validate values coming from the input. That's why it is readonly when running on the browser.
             target: { value: "01/12/2020" }
         });
-
         expect(setRangeValues).toBeCalledTimes(1);
     });
+
+    test.each([
+        ["d/m/yyyy", "dd/m/yyyy"],
```

Comment:
shouldn't the expected value be `dd/mm` instead of `dd/m`? Same for all the others where it's single `m`

Reply:
No, @tamelbrun said that Month is always in Capital letter.

Reply:
but only one? Your function is called `doubleMonthOrDayWhenSingle` 😄